lambda/get/main.go

```diff
@@ -38,7 +38,7 @@ func (l *Lambda) HandleEvent(ctx context.Context, event events.APIGatewayProxyRe
 
 	if err != nil {
 		l.logger.Print(err)
-		return shared.ProblemInternalServerError.Respond()
+		return shared.ProblemNotFoundRequest.Respond()
```

Comment:
Could we be converting a 500 response from the client into a 404, when in fact it's a legitimate 500? We should probably interrogate the request to work out what the status code, rather than always return a 404.